markdown/chapters/appendices/playables.md

```diff
@@ -0,0 +1,278 @@
+wording of it is a little restrictive. Making {U}{R} in a deck that wants to go
+from {B}{B}{B} to {U}{U} can be tricky, especially with so many mana denial
+elements omni-present in the format. The order of operations on the card also
+mean that if you are in a position to cast it with only two cards left in your
+deck, you would put one into your hand and one back into the deck but exile
+none. An excellent card for pre-combo play but not brilliant for after Doomsday
+resolves.
+
+## Mill Cantrips
+
+:::row{variant=centered}
+- Mental Note
+- Thought Scour
+:::
+
+Before :card[Consider] arrived on the scene, these were two options for being
+able to dig three cards deep into your pile with limited mana. They do have
+benefits similar to Consider such as being instant speed and thus allowing to
+dig three through :card[Narset, Parter of Veils] or :card[Spirit of the
+Labyrinth] however only allowing one card to be drawn means you have to have a
+good payoff to get to (something like :card[Unearth] for example having milled
+an Oracle) and unlike *Surveil*, (which Consider basically has) there is no
+*may* clause for the mill so if you get hit by a Surgical Extraction for example
+you could have your day ruined quite considerably. You also cannot use these
+cleanly in the same Brainstorm piles you can use Consider in because they need
+to have the single mill clause only to be effective. The cards are fine but
+honestly Consider is just better.
+
+## Echo of Eons
+
+:::row{variant=centered}
+- Echo of Eons
+:::
+
+Echo of Eons is one of the various *draw 7* effects that have existed in Magic
+and one of the few that are legal in Legacy. It can be a very powerful tool
+allowing for some unique pile construction that plays well around Endurance or
+makes good use of Lion's Eye Diamond. If you run a list with an excess of Ritual
+effects you can ply extra {B} into the *Flashback* cost for Echo and cards like
+Predict or Consider allow you to easily place it into the graveyard.
+
+You have to be wary though as you can deck yourself with the card if you
+miscount and forget that it exiles itself. There are also plenty of issues you
+could run into such as Hullbreacher that can punish such plays. This fits into
+the "probably a singleton for pile purposes but running a second Oracle likely
+does the same thing" category of cards.
+
+## Disruption and Removal
+
+:::row{variant=centered}
+- Drown in the Loch
+- Collective Brutality
+:::
+
+These cards can be very powerful allowing for a nice mix of flexibility and
+utility whilst being within the deck's core colours. The issue with them
+normally stems from the fact that two mana in Legacy is quite a lot and neither
+of these options presented are guaranteed to be effective in every matchup. If
+you are playing a slower, more controlling variant then they might have space as
+a one-of but I would not recommended running them.
```

Comment:
besides the mana costs, trading cards for creatures is often not a good deal for doomsday.
For hate pieces, or a Marit Lage, one of the common bounce spells is enough